library/core/src/main/java/com/google/android/exoplayer2/text/cea/Cea608Decoder.java

```diff
@@ -231,23 +236,41 @@
   private byte repeatableControlCc1;
   private byte repeatableControlCc2;
 
+  private int incomingDataTargetChannel;
```

Comment:
nit: Maybe just call it `currentChannel`? Not sure the extra words add anything :).

Reply:
You may rename at as you like during the pull, or I can upload a new revision with the new name as well.

Reply:
Ack; no need for you to update.